src/version.h

```diff
@@ -4,4 +4,4 @@
 
 	 NB: OPENQL_VERSION_STRING is also decoded by setup.py
 */
-#define OPENQL_VERSION_STRING "0.8.1.dcl-3"
+#define OPENQL_VERSION_STRING "0.8.1.dcl3"
```

Comment:
This is not a legal version for the module ("dcl" is not a recognized string)  and most probably what causes the CI to fail with ImportError. Legal names should follow this rule

[N!]N(.N)*[{a|b|rc}N][.postN][.devN]

according to https://www.python.org/dev/peps/pep-0440/

This is confirmed by compiling this PR locally and getting this warning:
<path>/python3.7/site-packages/setuptools/dist.py:484: UserWarning: The version specified ('0.8.1.dcl3') is an invalid version, this may not work as expected with newer versions of setuptools, pip, and PyPI. Please see PEP 440 for more details.
  "details." % self.metadata.version